src/components/forms/SccForm.js

```diff
@@ -68,6 +68,102 @@ class SccForm extends Component {
   render() {
     return (
       <>
+        <IcseFormGroup>
+          {/* group id */}
+          <IcseTextInput
+            id="scc_group_id"
+            tooltip={{
+              content: "The group ID for Security and Compliance Center.",
+            }}
+            componentName="SCC"
+            field="id"
+            labelText="Group ID"
+            value={this.state.id}
+            onChange={this.handleInputChange}
+            maxLength={255}
+            invalid={
+              invalidRegex(
+                "id",
+                this.state.scope_description,
+                this.props.descriptionRegex
+              ).invalid
+            }
+            invalidText={
+              invalidRegex(
+                "id",
+                this.state.scope_description,
+                this.props.descriptionRegex
+              ).invalidText
+            }
+          />
+          {/* collector-description text input */}
+          <IcseTextInput
+            id="scc_passphrase"
+            tooltip={{
+              content: "Security and Compliance Center group passphrase.",
+            }}
+            labelText="Group Passphrase"
+            field="passphrase"
+            value={this.state.passphrase}
+            onChange={this.handleInputChange}
+            componentName="SCC"
+            maxLength={1000}
+            invalid={
+              invalidRegex(
+                "passphrase",
+                this.state.passphrase,
+                this.props.descriptionRegex
+              ).invalid
+            }
+            invalidText={
+              invalidRegex(
+                "passphrase",
+                this.state.passphrase,
+                this.props.descriptionRegex
+              ).invalidText
+            }
+          />
+        </IcseFormGroup>
+        <IcseFormGroup>
+          <IcseNameInput
+            id={this.props.data.name + "-scc-name"}
+            componentName="scc-collector"
+            labelText="Collector Name"
```

Comment:
should collector name and description be side by side? 
<img width="501" alt="Screen Shot 2023-03-14 at 1 13 49 PM" src="https://user-images.githubusercontent.com/11401353/225125695-024cc671-4362-4119-a6f3-f40d7a160fc2.png">

Reply:
this is a typo, should be `Credential Name`